controllers/clusterversion_controller.go

```diff
@@ -436,3 +465,40 @@ func (c *ClusterVersionReconciler) ensureConsolePlugin() error {
 
 	return nil
 }
+
+func (c *ClusterVersionReconciler) reconcileCSIAddonsSubscription() error {
+	csiAddonsSubscription, err := c.getSubscriptionByPackageName(csiAddonsSubscriptionPackageName)
+	if err != nil {
+		return err
+
+	}
+
+	//We want the channel of csiAddons subscription to be stable-4.15 so no need to compare it with ocs-client-operator subscription
+	csiAddonsSubscription.Spec.Channel = "stable-4.15"
```

Comment:
you need to do update here which is missing

Reply:
Added

Reply:
Comment should be slightly rephrased, the reasoning we want stable-4.15 is because of a direct merge to release-4.15 channel and this code shouldn't be back ported and from 4.16 anyways the logic is already changed.

Reply:
Updated the comment to
```
//the channel of csiAddons subscription here is stable-4.15 because of a direct merge to release-4.15
//this code shouldn't be back ported and from release-4.16 the logic is changed
```